### PR TITLE
Update Dockerfile to pull latest stable Ubuntu. Small update to .yml file

### PR DESCRIPTION
**Pull request by TheSteveK**
When attempting to run `make docker_build` following readme.md, the build process was crashing due to the Dockerfile attempting to install Ubuntu Groovy, which has been [retired](https://fridge.ubuntu.com/2021/07/25/ubuntu-20-10-groovy-gorilla-end-of-life-reached-on-july-22-2021/)

Updated the release docker installs to the current stable release (Focal).

Also made a slight update to the YML file to use the new syntax for build context and file.

Testing instructions:

Follow the readme.md file and build a new docker instance of Crowdsignal, verify that there are no 404 errors.

### CODE REVIEW

**Comment by TheSteveK** [comment]
Ran into an error on an M1 mac: no matching manifest for linux/arm64/v8 in the manifest list entries when trying to spin up MySQL.  Added a line to the .yml file to force Docker to use the x64 version per https://stackoverflow.com/questions/65456814/docker-apple-silicon-m1-preview-mysql-no-matching-manifest-for-linux-arm64-v8

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -37,8 +40,8 @@ services:
       - "${PORT_CS_WORDPRESS:-8000}:80"
     restart: always
     extra_hosts:
-      - "api.crowdsignal.com:${CS_SANDBOX_IP:-192.0.123.248}"
-      - "app.crowdsignal.com:${CS_SANDBOX_IP:-192.0.123.248}"
+      - "api.crowdsignal.com:${CS_SANDBOX_IP:-192.0.123.178}"
```
Comment (CGastrell) [inline docker/docker-compose.yml:43]: These 2 lines need to remain as they were.

**Review by CGastrell — CHANGES_REQUESTED** [review]
Nice work! I left a comment on 2 lines and will test this further.

File: docker/docker-compose.yml (hunk 1 above)
Comment (TheSteveK) [reply docker/docker-compose.yml]: I've corrected those lines

**Review by CGastrell — APPROVED** [review]
Nice! Build and docker up worked perfectly. I was able to make the docker talk to my sandbox.
:shipit:

**Pull request merged**